cluster/aws/util.sh

```diff
@@ -47,6 +47,12 @@ MASTER_INTERNAL_IP=${INTERNAL_IP_BASE}${MASTER_IP_SUFFIX}
 MASTER_SG_NAME="kubernetes-master-${CLUSTER_ID}"
 MINION_SG_NAME="kubernetes-minion-${CLUSTER_ID}"
 
+# Be sure to map all the ephemeral drives.  We can specify more than we actually have.
+# TODO: Actually mount the correct number (especially if we have more), though this is non-trivial, and
```

Comment:
Is there an issue associated with this TODO? It seems like something that would be silently hit (and somewhat non-trivial to diagnose?).